AcademyChallenge/Data/Service/LiveAvatarStorage.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  LiveAvatarStorage.swift
+//  AcademyChallenge
+//
+//  Created by Keven Esmael on 03/10/2022.
+//
+
+import Foundation
+import UIKit
+import CoreData
+
+class LiveAvatarStorage {
```

Comment:
Why isn't the Storage class implementing the `AvatarService` protocol?

Reply:
solved, I didn't implement because I'm not using mocks